src/app/utils/field-validation/FieldValidatorFactory.class.js

```diff
@@ -41,8 +40,9 @@ class FieldValidatorFactory {
     // 'statement' and 'image' are rejected prior to the creation of a field validator
     switch (fieldType) {
       case 'section':
-        // Answers are disallowed for these fields
-        return new AnswerNotAllowedValidator(...arguments)
+      case 'textfield': // short text
+      case 'textarea': // long text
+        throw new Error(`${fieldType} has been migrated to TypeScript`)
```

Comment:
@karrui @mantariksh i added some new code to make sure there's no additional validation coming from the previous implemetation.